### ISSUE TITLE
Commando: More explicit command breadcrumbs

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#1420]
### Short Summary

I'd like to see a more detailed breadcrumb path of how I've arrived at the specific location in Commando.  The line that holds the command to be run seems to have enough space to cover this though I'm not that it could accommodate every possible path into commandos various scopes.  This could also be an additional line to hold the path of how the user arrived in that scope.

#### Benefits?

I see a few possible benefits to this:
1. similar to having the path to your current location in the terminal shown prepended to your input indicator `~/careyh/project/komodoide careyh$`.  My mac doesn't have it but I think Linux does  that?  Then i know how to get out to another scope without running pwd.  I'd have the same feeling in commando, knowing how to get back up to the appropriate level in Commando scopes to get where i want to be
2. Teach users how a button got them to a specific scope, eg. Clicking the NPM dynamic toolbar button then selecting Install Package takes you directly to the npm install scope but there is no indication to the user that they can get there themselves by opening commando and descending through `shell > npm > install`.  That could be awesome for getting users famliiar with such a massive tool as commando.

#### Cons?

None.  Maybe a chance for Carey to learn more about commando if he were to ve assigned this ticket.

**Comment by Naatan** [comment]
Could you give an example of what you see now and what you'd like to see?

**Comment by cgchoffman** [comment]
The path of how I arrive at where I am in commando, in the "command line" area.  There is the input field, then the part below where the command that will be run is held.  Right there is where I would want to see a trail of how I got to where I am.

**Comment by Naatan** [comment]
Oh you literally mean path as in directory path? I was thinking along the lines of a path that you walk .. :p Alright, I'll see about making this more evident.

**Comment by cgchoffman** [comment]
nono, not directory path.  I woudln't even know how that would work in Commando (when running a command).  I mean where I went in Commando to arrive at the current command.
eg. `Shell > Npm > Install`
A better example would be (arrived at through `Tools > Add-ons` :
What's shown:
`installed packages`
What i want to see:
 `Packages > manage packages > list installed`

**Comment by Naatan** [comment]
Ahh I getcha. In fact it does do that! I pushed a fix for this a few days ago. Try the latest nightly and let me know if that addresses this ticket.

**Comment by cgchoffman** [comment]
Ahhhh ok, installing and trying now....*time passes*...I think that fix was specifically for the #1419.  If I use my second example (`Tools > Add-ons`) then I still only see `installed packages` rather than `Packages > Manages Packages > Installed Packages`.

Also, with the fix for #1419, it does fix the commando but my suggestion is to provide more information in that field, how the user would get there when entering Commando on their own.  So now it shows `NPM > install` when I select `install package` from the NPM dynamic toolbar button.  My suggest is to prepend that with `shell` as well:
`shell > npm > install`.

**Comment by Naatan** [comment]
Right, but that's 2 enhancements in one. Lets keep this one about the path for now. I think it's best if you log a separate ticket for the "indicate how I got here" idea.

The reason you don't see "Packages" is because Packages is a scope, not a subscope. But that's a technical reason and not a UX reason; so I'll leave this ticket open to address this.

**Comment by cgchoffman** [comment]
I don't understand.  I'm only asking for one thing.  To be shown where I am in Commando.  I don't see two enhancement requests.  What is the second one?

**Comment by Naatan** [comment]
I thought that nr 2 was about showing some sort of animation to indicate the process, or did you mean simply that showing the full path should properly convey how we got there?

**Comment by cgchoffman** [comment]
No animation, no.  Simply showing the entire path of how you would get there if you were to enter Commando manually through Ctrl +  Shift + O.  Those numbered items in the description are just pros of the one enhancement request.

**Comment by Naatan** [comment]
Alright gotcha. Nevermind the second ticket then :)
